tests/supply_chain/test_action_reset.py

```diff
@@ -0,0 +1,790 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Both file `test_action_reset` and `test_env_reset` in this PR, prefer the naming `test_env_reset`